[W-10728932] New landing page (#120)
mulesoft/docs-general: modules/ROOT/pages/11-2022-index.adoc

--

image::discover.png[alt=" "]

--
[discrete]
=== Discover

Browse Exchange to find existing API specifications, templates, examples, and other assets that you can reuse for your projects.

* https://www.anypoint.mulesoft.com/exchange/[Discover reusable assets.^]
--

image::design.png[alt=" "]

--
[discrete]
=== Design

Create and publish new API specifications from scratch.

* xref:design-center::design-create-publish-api-specs.adoc[Create a reusable API specification.]
* xref:access-management::index.adoc[Manage access to your Anypoint Platform account.]
* xref:general::security.adoc[Build security into your application network.]
--

image::develop.png[alt=" "]

--
[discrete]
=== Develop & Test

Build and test APIs and integration apps.

* xref:studio::index.adoc[Develop your API.]
* xref:connectors::index.adoc[Connect your data, systems, and apps.]
* xref:mule-runtime::mule-app-dev.adoc[Develop the Mule app that integrates systems, services, APIs, and devices.]
* xref:dataweave::index.adoc[Access and transform data within the Mule app.]
* xref:munit::index.adoc[Build automated tests for your APIs and integrations.]
--

image::deploy.png[alt=" "]

--
[discrete]
=== Deploy

> Deploy APIs and apps to your hosting targets, and secure them with policies.

Reviewer: ```suggestion
Choose a deployment option, deploy APIs and apps, and secure them with policies.
```
